java assignments/assignments/assignment2/SearchAlphabets.java

```diff
@@ -0,0 +1,40 @@
+package com.assignments.assignment2;
+
+import java.util.Scanner;
+
+public class SearchAlphabets {
+    public static void main(String args[])
+    {
+        Scanner sc=new Scanner(System.in);
+        System.out.println("enter the string to search");
+        String string=new String(sc.nextLine().toLowerCase());
+        byte b[]=string.getBytes();
```

Comment:
this can be done without converting into bytes also please check once.

Reply:
i have added the alternate method too and commented that code